src/watermark.plugin.js

```diff
@@ -0,0 +1,60 @@
+function WikiaJWPlayerWatermarkPlugin(player, config, div) {
+	this.player = player;
+	this.container = div;
+	this.config = config;
+	this.watermarkElement = this.getWatermarkElement();
+	this.watermarkElement.addEventListener('click', function () {
+		player.trigger('watermarkClicked');
+	});
+
+	this.container.classList.add('wikia-watermark-container');
+
+	this.isEnabled = config.show;
```

Comment:
I think this should be the value of `watermark` custom parameter from the current video.